2022/_includes/diversity/diversity-terms.html

```diff
@@ -7,33 +7,45 @@
 People who are diverse in nationality, age, race, ethnic origin, spiritual beliefs and practices, gender, gender identity, sexual orientation, work and life status are critical for constant improvement and innovation in our profession.</p>
 </div>
 <div class="paragraph">
-<p>That why at JBCNConf we are offering diversity a plan.</p>
+<p>That&#8217;s why we are running again the <em>diversity a plan for JBCNConf 2022</em> 🎉</p>
 </div>
 <div class="paragraph">
 <p>If you belong to a traditionally underrepresented group in the tech community, now is your chance.
-We definitely want you to come and join us at the conference.</p>
+We definitely want you to come and join us at the conference 🫂</p>
```

Comment:
```suggestion
We definitely want you to come and join us at the conference</p>
```

Reply:
Lo de la sugestión es automático?

Reply:
Interestingly, it does work in MacOS. The thing is that was the only "hug-related" neutral emoji I saw, other than removing, someone has any suggestion?

Reply:
I just updated my commit suggestion  with the hug emoji from https://emojipedia.org/hugging-face/ you can look for any other as well

Reply:
Thats the thing, already did and that one points to and does not work with my Linux fonts
![image](https://user-images.githubusercontent.com/5781153/157645393-ac15a1c8-5202-4a43-b890-f83129c19335.png)...I love multi-platforms 🌈

Reply:
A falta de otro, aplico 🤗